config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```diff
@@ -54,6 +54,54 @@ presubmits:
             cpu: "2"
             memory: "6Gi"
 
+  # This jobs runs e2e.test with a focus on tests for the VolumeAttributesClass feature (currently alpha)
+  # on a kind cluster
+  - name: pull-kubernetes-e2e-storage-kind-vac-feature
```

Comment:
Does SIG Storage have other features that need a special cluster?

It might make more sense to have a `pull-kubernetes-e2e-storage-kind-alpha-features` job where SIG storage alpha feature gates + API groups are enabled and then all tests for those run. Because this is a job owned by the SIG, you know when to update this job during feature promotion.

Reply:
I don't know but I believe that the answer will be yes. Perhaps we need the feature owner to complete this section as well when they add e2e tests.

Reply:
even if the feature is promoted to beta, the cluster created by the alpha job still turns on the feature gate regardless of what the feature-gate stage is.

So I want to rename the job to `pull-kubernetes-e2e-storage-kind-alpha-beta-features` in order to void the confusion.

cc @pohly

Reply:
That name works for me.

We still have the confusion between "job that tests alpha/beta features" (this one here) vs. "job which tests GA features with alpha/beta features enabled" (the SIG Testing jobs), but I can't think of better names.